packages/gatsby-source-contentful/src/__tests__/gatsby-node.js

```diff
@@ -564,7 +564,7 @@ describe(`gatsby-node`, () => {
           "Contentful: 0 deleted entries",
         ],
         Array [
-          "Contentful: 11 cached entries",
+          "Contentful: 4 cached entries",
```

Comment:
I don't quite get why this changed? And if this is actually correct? Those things are logged before we do any processing (?)

Reply:
This is because the JSON and Text nodes no more have the `sys.type = "Entry"`.

https://github.com/gatsbyjs/gatsby/pull/35214/files#diff-2b62a402aa215f93622f01ba9a39744cb20282ce034d2d87c177fd3d89f9c120R200

Will filter them now out of the counting.